app/Console/Commands/TeamSpeak/TeamSpeakDaemon.php

```diff
@@ -41,13 +41,26 @@ class TeamSpeakDaemon extends TeamSpeakCommand
     public function handle()
     {
         self::$connection = $this->establishConnection();
+        $connection_failures = 0;
```

Comment:
Needs amending!

Reply:
That's what I get for using the Git IDE :/